New package: LandSea v0.0.1

JuliaRegistrator opened the pull request:
- Registering package: LandSea
- Repository: https://github.com/GeoRegionsEcosystem/LandSea.jl
- Created by: @natgeo-wong
- Version: v0.0.1
- Commit: 1e8ca072f452703a93c696e868b540b8d7d6f14d
- Reviewed by: @natgeo-wong
- Reference: https://github.com/GeoRegionsEcosystem/LandSea.jl/commit/1e8ca072f452703a93c696e868b540b8d7d6f14d#commitcomment-150867391
- Release notes:
<!-- BEGIN RELEASE NOTES -->
`````
* Initial release of LandSea.jl, very bare bones, smoothing functionalities to come. Documentation will be released at a later date.
`````
<!-- END RELEASE NOTES -->

<!-- bf0c69308befbd3ccf2cc956ac8a46712550b79fc9bfb5e4edf8f833f05f4c18b06eddad8845b45beb9f45c2b8020dd612faeca880bc6623d73ab5d481f549e7666ee5a21807abab425339ee45b1c6fcb5549e1acefbcd730f0bea3a961ec8a1b13920296c7b0e98ddd7f8369091b5b658d16d3459fd2a1e98f2e2854c733f4089aebe482a79abb4d3269ce7002b309f22b31dc38dd6394ce5ce3c3afe308e4573d60b5db5b459ab3a01f61fe30b72180546ac5f9db195d28543b64a3e50b69cd960e3a4943022b0b7904a28c8cbf5f82ed39a056593940debf9f774d8529c8d -->

goerz commented:
Thank you for submitting your package! Could you fix the links from the README to the documentation?

natgeo-wong commented:
[noblock] alright a basic docs link should be up

JuliaRegistrator committed:
New package: LandSea v0.0.1

UUID: cbf7bb22-a1a6-46cb-8b24-174e3bc5fc04
Repo: https://github.com/GeoRegionsEcosystem/LandSea.jl.git
Tree: 2f3dcf176258981df7d2ae323b1d652b1f33b803

Registrator tree SHA: 17aec322677d9b81cdd6b9b9236b09a3f1374c6a
